ckanext/validation/tests/logic/test_action.py

```diff
@@ -0,0 +1,465 @@
+# encoding: utf-8
```

Comment:
could you just leave this as test_logic.py so that it can be delta'd easier. Major movement/refactoring should not be done till things have stabilised or prior to alterations.

Reply:
\+